tests/playwright/sanity/container.test.js

```diff
@@ -487,7 +481,7 @@ test.describe( 'Container tests', () => {
 		// Assert
 		expect( await container.screenshot( {
 			type: 'jpeg',
-			quality: 70,
+			quality: 90,
 		} ) ).toMatchSnapshot( 'container-column-spacer-align-center.jpeg' );
```

Comment:
Can we skip the tests with the 'row' styling for now?
Can we remove the incorrect screenshots with the missing divider widget, or how would you like to go about that?

Reply:
I want it to fail when we fix the issue , this will force us to fix things now and not create another task for the future